main_console.js

```diff
@@ -0,0 +1,126 @@
+var startBoard = [
+  [1, 2, 3],
+  [4, 5, 6],
+  [7, 8, 9]
+];
+var board = [];
+var player1 = "X";
+var player2 = "O";
+var messages = {
+  welcome: "Welcome to Tic Tac Toe!",
+  player1First: "Player One goes first! You are 'X'. Choose an unoccupied position from the board!",
+  player2First: "Player Two goes next! You are 'O' Choose an unoccupied position from the board!",
+  player1Next: "Player One's turn again!",
+  player2Next: "Player Two's turn again!",
+  wrongMove: "That move is not allowed, try again."
+}
+var message = "";
+var player = "";
```

Comment:
What's the difference between `player1`, `player2` and `player`. Often times when variable names are confusing like this, it either means the names need to be altered to be more specific, or something could be buttoned up a bit.